Signal/translations/en.lproj/Localizable.strings

```diff
@@ -943,6 +958,9 @@
 /* Label indicating that the user is not verified. Embeds {{the user's name or phone number}}. */
 "PRIVACY_IDENTITY_IS_NOT_VERIFIED_FORMAT" = "%@ is not verified.";
```

Comment:
Maybe better copy would be: "You have not verified Bob." or  even "You have not marked Bob as verified."

The confusion I want to avoid is the perception that "verified" is some service level attribute. Rather "verification" is a tool for Alice to classify each of her contacts.

Contrast this with the completely different twitter concept of "verified". When Bob has a "verified" twitter account, it means Bob took some action, then  appears "verified" to *all* twitter users. Alice had no role in that. I think the recommended copy better distinguishes what we're doing from a user's potentially previous experiences.

WDYT @charlesmchen / @riyapenn